projects/user-portal/src/app/components/request/request.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="buttons">
+  <button class="button" (click)="backToRoom()">Back to room #{{roomNumber}}</button>
+  <button class="button" (click)="backToFloor()">Back to floor #{{floorNumber}}</button>
+</div>  
+<div class="request" >
+  <div class="request__info">
+    <p class="request__static-text">Request <b class="request__info-text">#{{requestInfo.id}}</b></p>
+    <p class="request__static-text">Status - <b class="request__info-text">{{requestInfo.status | requestStatus}}</b></p>
+    <p class="request__static-text">Request Owner - <b class="request__info-text">{{requestInfo.requestOwner}}</b></p>
+    <p class="request__static-text">Open at <b class="request__info-text">{{requestInfo.openDate | date:'short'}}</b></p>
+    <button class="request__button" (click)="closeRequest()">Close Request</button>
+  </div>
+  <div class="request__comments comments">
+    <div class="comments__comment-form comment-form">
+      <textarea [formControl]="newComment" class="comment-form__text"></textarea>
+      <p class="comment-form__title">Post comment as:</p>
+      <div class="comment-form__buttons" *ngIf="requestInfo.status === 2 || requestInfo.status === 3">
+        <button class="button" (click)="sendReOpenComment()">Reopen</button>
+        <button class="button" (click)="sendCommentWithoutStatus()">Without status</button>      
+      </div>
+      <div class="comment-form__buttons" *ngIf="requestInfo.status !== 2 && requestInfo.status !== 3">
+          <button class="button" (click)="sendCommentWithoutStatus()">Without status</button>      
+      </div>
+    </div>
+    <div class="comments__comment comment" *ngFor="let comment of requestInfo.comments">
```

Comment:
You can add trackby function to decrease rerendering. In your case it won't increase performance, but in some cases it is vital